.travis.yml

```
@@ -1,3 +1,16 @@
+language: ruby
+cache: bundler
+rvm:
+  - 2.7.1
```

```suggestion
  - 2.7
```

We probably don't need to specify the version number so strictly